todo-list-backend/tests/integration/graphql-api.spec.ts

```diff
@@ -447,10 +534,13 @@ describe("mutate", () => {
 
         describe("updateTodoName", () => {
             it("raises an error", async () => {
+                let userWithTodo = await storage.readUser("nora");
+                let todoToChange = (await storage.readTodos(userWithTodo))[0];
+
                 let response = await mutate({
                     mutation: UPDATE_TODO,
                     variables: {
-                        id: storage.readTodos(storage.readUser("nora"))[0].id,
+                        id: todoToChange.id,
                         name: "new todo name"
                     }
                 });
```

Comment:
könnte man auch in ein expect packen:

await expect(mutate({ mutation: UPDATE_TODO , variables })).resolves.toMatchObject({
  data: { updateTodo: null },
  errors: {message: "Not Authorised!"}
})